src/node_crypto.cc

```diff
@@ -745,6 +746,24 @@ void SecureContext::SetKey(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+void SecureContext::SetSigalgs(const FunctionCallbackInfo<Value>& args) {
+  SecureContext* sc;
+  ASSIGN_OR_RETURN_UNWRAP(&sc, args.Holder());
+  Environment* env = sc->env();
+  ClearErrorOnReturn clear_error_on_return;
+
+  CHECK(args.Length() == 1 && args[0]->IsString());
```

Comment:
Can you make these separate checks and use `CHECK_EQ(1, args.Length())` for the first one?